operations/service_web.go

```diff
@@ -65,6 +65,8 @@ func startWebService() cli.Command {
 
 			tracer := tp.Tracer("github.com/evergreen-ci/evergreen/operations")
 			ctx, startServiceSpan := tracer.Start(ctx, "StartService")
+			// This is only in case of an error.
```

Comment:
we might not need this comment (unless you want to keep it!)

Reply:
I think keep it since it might be non-obvious why we close the span twice (which does nothing by otel standards, the 2nd call will do no-op)